tests/test_simresults.py

```diff
@@ -22,7 +22,7 @@
 from pulser import Pulse, Register, Sequence
 from pulser.devices import Chadoq2, MockDevice
 from pulser.waveforms import BlackmanWaveform
-from pulser_simulation import SimConfig, Simulation
+from pulser_simulation import QutipEmulator, SimConfig, Simulation
```

Comment:
Where is `Simulation` still being used here? 
If it's not being used, why doesn't `flake8` complain?
If it is, why doesn't it raise the `DeprecationWarning`?

Reply:
It was still being used in a type declaration in a test function. The type was wrong (saying that fixture `sim` is a Simulation whereas it is a `QutipEmulator`) so I guess `mypy` should have complained but I don't think it checks the tests.

Reply:
Yeah, it doesn't, type hints are mostly used to help the IDE auto-complete